test/plots/difference.ts

```diff
@@ -0,0 +1,13 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export async function differenceY() {
+  const aapl = await d3.csv<any>("data/aapl.csv", d3.autoType);
+  const goog = await d3.csv<any>("data/goog.csv", d3.autoType);
+  const x = aapl.map((d) => d.Date);
+  const y1 = aapl.map((d, i, data) => d.Close / data[0].Close);
+  const y2 = goog.map((d, i, data) => d.Close / data[0].Close);
```

Comment:
It makes sense to normalize, but the chart is difficult to read: which part is "goog > aapl" and vice-versa? Maybe we can solve it by adding annotations, rather than a color legend?

Reply:
I also want to try comparing week-over-week or month-over-month change. This might involve shifting _x1_ or _x2_ instead of separating _y1_ and _y2_. We should test this use case.